mailpoet/lib/EmailEditor/Engine/SettingsController.php

```diff
@@ -30,7 +30,7 @@ class SettingsController {
    * Color of email layout background.
    * @var string
    */
-  const EMAIL_LAYOUT_BACKGROUND = '#cccccc';
+  const EMAIL_LAYOUT_BACKGROUND = '#f0f0f0';
```

Comment:
This is the color used in the designs.

Reply:
Thanks for the comment, I suppose we will change it soon with color styles.